frontend/lib/util/android_utils.dart

```diff
@@ -12,3 +12,13 @@ Future<bool> certificateIsRequired() async {
   AndroidDeviceInfo androidInfo = await deviceInfo.androidInfo;
   return androidInfo.version.sdkInt < 25;
 }
+
+Future<bool> isDeviceWithCameraIssues() async {
+  if (!Platform.isAndroid) {
+    return false;
+  }
+  List<String> devicesWithoutQRCodeDetection = ['SM-A236B'];
```

Comment:
There are more model IDs for the Samsung Galaxy A23. We should probably take all listed here:
https://storage.googleapis.com/play_public/supported_devices.html

I think as a workaround it's fine if it does not look great on these devices, but at least works well enough.

Have you checked whether a higher resolution would work too?

Reply:
Well it doesn't matter what resolution you take it's just important to set a default resolution. It will not really be used. I mean you can check it on your device by adding your model and test it. It's just changing the aspect ratio which looks weird but for some devices its ok-

I can add these devices, even we don't really know it they also cause this issue. Were only reported for A23 5G.
It would be really better to maintain that list somewhere where we could add it without triggering new release.

Reply:
There are ~10 model IDs for the A23 5G.

Not sure if it's worth the effort to do more engineering around this workaround. We should rather try to fix the upstream issue if we want to invest more time :D